SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -105,7 +105,7 @@ public string LoadAndFixFile(string sarifPath, ILogger logger)
             if (!File.Exists(sarifPath))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) In member 'RoslynV1SarifFixer.LoadAndFixFile(string, ILogger)', change parameter name 'sarifPath' to 'sarifFilePath' in order to match the identifier as it has been declared in 'IRoslynV1SarifFixer.LoadAndFixFile(string, ILogger)'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AParameterNamesShouldMatchBaseDeclaration)